src/k8s/variables.tf

```
@@ -154,4 +154,9 @@ variable "configmaps_bpdmswinningtransaction" {
 
 variable "configmaps_rtdpaymentinstrumentmanager" {
   type = map(string)
+}
+
+
+variable "configmaps_facustomer_override" {
```

```suggestion
variable "configmaps_facustomer" {
```
for naming convention